apps/docs/src/content/docs/explainers/mid-run-cancellation.mdx

```diff
@@ -185,14 +184,14 @@ generator.next('ping');
 // logs: "after yield: ping"
 ```
 
-I know, I told you this wouldn't be scary and for the moment I haven't keep my promise (pun
-intended). But the main takeaway from this snippet of code is to show that generator functions have
+I know, I told you this wouldn't be scary and for the moment I haven't kept my promise (pun
+intended). But the main takeaway from this snippet of code is that generator functions have
 a way to stop executing and return something to the caller and the caller has a way to communicate
 something back.
 
 `@sheepdog/svelte` has been built to be able to accept an async generator function and, most
 importantly, has been built to make the generator function work basically like a normal async
-function if you change `await` with `yield`. Let's take a look
+function if you replace `await` with `yield`. Let's take a look
 
 <Tabs>
```

Comment:
The following tabs section also has this simple example where the `signal` already does all the work.